libs/output-mapping/src/Storage/StoragePreparer.php

```diff
@@ -16,6 +16,7 @@ class StoragePreparer
     public function __construct(
         readonly private ClientWrapper $clientWrapper,
         readonly private LoggerInterface $logger,
+        readonly bool $hasNewNativeTypeFeature,
```

Comment:
V minulym PR jsem to vyhodil, protože se to k ničemu nepoužívalo.https://github.com/keboola/platform-libraries/pull/285#discussion_r1665877509

 No teď se mi to hodí, tak jsem to přidal zase zpátky

Reply:
s tou hasChanges metodou mi to přišlo lepší a kdyby si v ní zkontroloval i PKs tak by "hezčí"